src/main.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+ class Person {
+    constructor (name, dateOfBirth, amount) {
+        this.name = name;
+        this.dateOfBirth = dateOfBirth.split('.');
+        const [day, month, year] = this.dateOfBirth;
+        this.amount = amount;
+        this.currentAmount = this.amount;
+        this.accountHistory = [{'initial': amount}];
+
+        this.age = function() {
+            const currentYear = new Date().getFullYear();
+            const currentMonth = new Date().getMonth() + 1;
+            const currentDay = new Date().getDate();
+            if (currentMonth < month || currentMonth === month && currentDay < day) {
+                return currentYear - year + 1;
+            }
+            return currentYear - year;
```

Comment:
you can create a variable for `currentYear - year` and return variable or variable + 1;